src/lib/ui/noise-sensor/NoiseSensorActivityList.tsx

```diff
@@ -0,0 +1,30 @@
+import type { NoiseSensorDevice } from 'seamapi'
+
+import { useEvents } from 'lib/seam/events/use-events.js'
+import { NoiseSensorEventItem } from 'lib/ui/noise-sensor/NoiseSensorEventItem.js'
+
+const since = new Date(Date.now() - 30 * 24 * 60 * 60 * 1000)
```

Comment:
This must `useCurrentTime`, see e.g., https://github.com/seamapi/react/pull/378#discussion_r1308080619

In this case, when should the result update? This implementation says "whenever the module is initialized" which is indeterminate behavior. The component should get the current time with `useCurrentTime` and then be explicit about when the time will update in the local component state (and thus refetch the query). 

What do you think is the right behavior here for the consumer?

Reply:
Good point! I'm not seeing the `useCurrentTime` hook in the repo currently. Was it removed at some point? I see it was implemented in #378 as mentioned.

Reply:
> What do you think is the right behavior here for the consumer?

Yeah, this is a good question. I mean, we only need to set the `since` parameter once when the component is first rendered, because all calls afterward will just pull from that range. I definitely need to set a `refetchInterval`, however. That could be 15-30 seconds or so for now.

Reply:
If we're only setting the `since` param on initialization, is the usage of `useCurrentTime` necessary in that case?

Reply:
Ah, I guess it's called `useNow` https://github.com/seamapi/react/blob/main/src/lib/ui/use-now.ts

You can set a `refetchInterval`, but in order for the UI to load more events without losing the exiting ones, we should bind the time to the state so it's only updated on unmount / remount.

```ts
const since = useState(new Date())
```